<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>WebBox</class>
 <widget class="QWidget" name="WebBox">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>320</width>
    <height>480</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>MiniWeb View</string>
  </property>
  <widget class="QWebView" name="webView">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>320</width>
     <height>480</height>
    </rect>
   </property>
   <property name="url">
    <url>
     <string>about:blank</string>
    </url>
   </property>
  </widget>
  <widget class="QPushButton" name="theButton">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>80</width>
     <height>12</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QGroupBox" name="newUrl">
   <property name="geometry">
    <rect>
     <x>25</x>
     <y>190</y>
     <width>256</width>
     <height>101</height>
    </rect>
   </property>
   <property name="title">
    <string>New Url</string>
   </property>
   <widget class="QLineEdit" name="textEnter">
    <property name="geometry">
     <rect>
      <x>5</x>
      <y>20</y>
      <width>246</width>
      <height>31</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="textOK">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>60</y>
      <width>83</width>
      <height>28</height>
     </rect>
    </property>
    <property name="text">
     <string>OK</string>
    </property>
   </widget>
   <widget class="QPushButton" name="textCancel">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>65</y>
      <width>83</width>
      <height>28</height>
     </rect>
    </property>
    <property name="text">
     <string>Cancel</string>
    </property>
   </widget>
  </widget>
 </widget>
 <layoutdefault spacing="0" margin="0"/>
 <customwidgets>
  <customwidget>
   <class>QWebView</class>
   <extends>QWidget</extends>
   <header>QtWebKit/QWebView</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
